Aptoide/aptoide-client-v8
# [APP-2451] - update bottom bar, top bar and toolbar UI

**What does this PR do?**

 It updates the bottom bar, top bar and toolbar UI for aptoide games according the design
 
 (It's currently missing the bodyBold font, I believe)

**Database changed?**

No

**Where should the reviewer start?**


**How should this be manually tested?**

Check every use case on the different bars and test them.
  Flow on how to test this or QA Tickets related to this use-case: [APP-2451](https://aptoide.atlassian.net/browse/APP-2451)

**What are the relevant tickets?**

  Tickets related to this pull-request: [APP-2451](https://aptoide.atlassian.net/browse/APP-2451)

**Questions:**

   Does this add new dependencies which need to be added? (Eg. new keys, etc.) 



**Code Review Checklist**


[APP-2451]: https://aptoide.atlassian.net/browse/APP-2451?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[APP-2451]: https://aptoide.atlassian.net/browse/APP-2451?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
